plugins/modules/mail.py

```diff
@@ -246,6 +246,7 @@ def main():
             subtype=dict(type='str', default='plain', choices=['html', 'plain']),
             secure=dict(type='str', default='try', choices=['always', 'never', 'starttls', 'try']),
             timeout=dict(type='int', default=20),
+            message_id=dict(type='str', aliases=['Message-ID'])
```

Comment:
Please use `message_id` without an alias.
```suggestion
            message_id=dict(type='str'),
```
Also you need to document this new option in `DOCUMENTATION` as the top of the file.